### ISSUE TITLE
Node sequence >2 do not render

### ISSUE THREAD

**Issue by Obsidian-user** [javierluraschi/nomnoml#30]
Creating a sequence of nodes longer than 2 does not render at all in either the `nomnoml` chunk or on document render.

Taking the first line from `nomnoml.com` as an example; it renders on the webpage but only renders a blank page in RStudio:

```
[<start>start] -> [<state>plunder] -> [<choice>more loot] -> [start]
```

Perhaps this is intended behaviour as one of your basic examples breaks a simple sequence into two separate lines; `[a]->[b]->[c]`. I thought I'd mention it as I couldn't find it in the issue tracker and it might trip someone else up.

Thanks for all your work team.

**Comment by andrie** [comment]
Thanks.  It seems that `nomnoml.js` is now using a new parser.  If you click the "legacy parser" on the main site you'll find the same problem.

We'll have to figure out how to opt into the new parser inside using the `htmlwidget`.

![image](https://github.com/rstudio/nomnoml/assets/479998/e6a92fe5-4841-4ffd-85bc-04904d0e9efe)

**Comment by andrie** [comment]
This is now fixed in the dev version.

**Comment by andrie** [comment]
Fixed in 76d609d288276c7712f590d7517eb56cdbb72286
